### PR TITLE
[GH-634] - Add traps documentation

### LINKED ISSUE #634: [DOCS] - Add traps documentation

### PR DESCRIPTION
**Pull request by tvillegas98**
## Motivation

Add documentation regarding the new implemented feature "Traps"
Closes #634

## Summary of changes

- Added a markdown explaining what traps are and their fields

## How to test it?

Read it (?)

## Checklist
- [ ] Tested the changes locally.
- [x] Reviewed the changes on GitHub, line by line.
- [x] This change requires new documentation.
  - [x] Documentation has been added/updated.

### CODE REVIEW

File: apps/arena/docs/src/attributes/traps.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Trap Attributes
+
+## Non-Changeable Attributes
+
+- `id`: Unique identifier for the projectile
```
Comment (agustinesco) [inline apps/arena/docs/src/attributes/traps.md:5]:
```suggestion
- `id`: Unique identifier for the trap
```

File: apps/arena/docs/src/attributes/traps.md
Comment (agustinesco) [inline apps/arena/docs/src/attributes/traps.md:1]: Can you put an example of a trap pls?

Comment (tvillegas98) [reply apps/arena/docs/src/attributes/traps.md:1]: Sure!

File: apps/arena/docs/src/attributes/traps.md (hunk 1 above)
Comment (tvillegas98) [reply apps/arena/docs/src/attributes/traps.md]: Whoops!

**Review by agustinesco — DISMISSED** [review]

File: apps/arena/docs/src/attributes/traps.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Trap Attributes
```
Comment (Nico-Sanchez) [inline apps/arena/docs/src/attributes/traps.md:1]: Can you explain shortly what a trap is, and what's their function?

Comment (tvillegas98) [reply apps/arena/docs/src/attributes/traps.md:1]: Sure!

**Review by Nico-Sanchez — APPROVED** [review]

**Pull request merged**